serving.proto

```diff
@@ -112,9 +122,15 @@ message SummarizationRequest {
     // Higher values penalize new tokens based on whether they appear in the text so far,
     // increasing the model's likelihood to talk about new topics.
     optional float presence_penalty = 20;
+    // This feature is in Beta. If specified, the LLM will make a best effort to sample deterministically,
+    // such that repeated requests with the same seed and parameters should return the same result.
+    // Note:- Seed value is only used when temperature is set to 0.
+    optional uint32 seed = 25;
   }
   ModelParams model_params = 215;
 
+  // Parameters for the citation style.
+  CitationParams citation_params = 220;
```

Comment:
Not all prompts support citation prompts. @pwoznic we should have a list somewhere in the docs that lists which prompts support this.

Reply:
The [citations content](https://docs.vectara.com/docs/api-reference/search-apis/search#citation-format-in-summary) lists the supposed prompts (5 at the moment).